src/VisualStudio/Core/Test.Next/Services/SolutionAssetCacheTests.cs

```diff
@@ -20,6 +20,12 @@ namespace Roslyn.VisualStudio.Next.UnitTests.Remote
     [Trait(Traits.Feature, Traits.Features.RemoteHost)]
     public class SolutionAssetCacheTests
     {
+        private static void ForceGC()
+        {
+            for (var i = 0; i < 3; i++)
+                GC.Collect();
```

Comment:
If this is being used for testing it's odd it isn't doing the WaitForPendingFinalizers that is often used....

Reply:
Yes.  thsi is very odd.